src/Core/Hook/Provider/GridDefinitionHookByServiceIdsProvider.php

```diff
@@ -133,7 +133,9 @@ private function getGridDefinitionIds(array $gridDefinitionServiceIds)
                 $camelizedDefinitionId = Container::camelize($definitionId);
                 yield $camelizedDefinitionId;
             } catch (Exception $e) {
-                Logger::addLog(sprintf('Error while loading service: %s . Error: %s', $serviceId, $e));
```

Comment:
why not throw the exception again if php in cli mode is detected so we can catch later in the command and display it the right way ?